css/styles.css

```diff
@@ -108,3 +164,209 @@ body {
   display: flex;
   padding: 1px;
 }
+
+.work-section {
+  display: flex;
+  padding-top: 40px;
+  justify-content: center;
+}
+
+.title {
+  font-family: 'Crete Round', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  font-size: 40px;
+  line-height: 52px;
+  margin-left: 24px;
+  border-bottom: 1px solid rgb(0, 0, 0);
+}
+
+.my-works {
+  display: flex;
+  flex-direction: column;
+  padding: 3px;
+  padding-left: 15px;
+}
+
+.work-pic-1 img {
+  margin-top: 50px;
+  margin-bottom: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  display: block;
+}
+
+.work-content {
+  grid-row: 2/3;
+  grid-column: 1/4;
+  font-size: 10px;
+}
+
+.work-content h2 {
+  font-family: 'Crete Round', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+  margin-left: 16px;
+}
+
+.work-content p {
+  font-family: 'Inter', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  padding-left: 16px;
+}
+
+.programming-languages {
+  display: flex;
+  align-items: left;
+  padding: 3px;
+  grid-gap: 5px;
+}
+
+.programming-languages p {
+  font-family: 'Inter', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  text-align: center;
+  height: 30px;
+  flex: none;
+  padding-left: 16px;
+  justify-content: space-between;
+  border: 1px solid black;

```

Comment:
Please remove the border to stick to hte Template
<img width="654" alt="image" src="https://user-images.githubusercontent.com/7243132/170840386-286df49f-38bd-4971-8215-9afa3fbfb8e1.png">